tests/system/action/projector/test_project.py

```diff
@@ -52,132 +53,166 @@ def test_project(self) -> None:
+        self.assert_model_exists("projection/105", {'content_object_id': 'assignment/452', 'history_projector_id': 23, 'stable': False, 'weight': 2})
+        self.assert_model_exists("projection/106", {'content_object_id': 'assignment/452', 'current_projector_id': 23, 'stable': True})
 
-    def test_project_not_unique_ids(self) -> None:
+    def test_try_to_store_second_stable_projection(self) -> None:
         response = self.request(
             "projector.project",
             {
-                "ids": [23, 23],
-                "content_object_id": "assignment/453",
-                "options": {},
-                "stable": False,
-                "type": "test",
+                "ids": [23],
+                "content_object_id": "assignment/452",
+                "meeting_id": 1,
+                "stable": True,
             },
         )
-        self.assert_status_code(response, 400)
-        assert "data.ids must contain unique items" in response.data.decode()
+        self.assert_status_code(response, 200)
+        self.assert_model_exists("projector/23", {'current_projection_ids': [105, 106, 112], 'scroll': 80})
+        self.assert_model_exists("projection/112", {'content_object_id': 'assignment/452', 'current_projector_id': 23, 'stable': True})
+        self.assert_model_exists("projection/105", {'content_object_id': 'assignment/452', 'current_projector_id': 23, 'stable': False})
+        self.assert_model_exists("projection/106", {'content_object_id': 'assignment/452', 'current_projector_id': 23, 'stable': True})
 
     def test_meeting_as_content_object_ok(self) -> None:
         response = self.request(
             "projector.project",
             {
-                "ids": [65],
+                "ids": [],
                 "content_object_id": "meeting/1",
-                "stable": True,
-                "type": "current-list-of-speakers",
+                "meeting_id": 1,
             },
         )
         self.assert_status_code(response, 200)
-        self.assert_model_exists(
-            "projector/65", {"current_projection_ids": [112], "meeting_id": 1}
+
+    def test_user_as_content_object_okay(self) -> None:
+        self.create_model("user/2", {"username": "normal user", "group_$1_ids": [1], "group_$_ids": ["1"]},)
+        response = self.request(
+            "projector.project", {"ids": [], "content_object_id": "user/2", "meeting_id": 1}
         )
-        self.assert_model_exists(
-            "projection/112",
-            {
-                "content_object_id": "meeting/1",
-                "current_projector_id": 65,
-                "meeting_id": 1,
-            },
+        self.assert_status_code(response, 200)
+
+    def test_temporary_user_as_content_object_okay(self) -> None:
+        """
+        Although the temporaray user has to have a group, this test checks the temporary user without group set
```

Comment:
So you are relying on `assert_belongs_to_meeting` to handle this, right? I would propose to add a group - maybe the behavior of it or some permission checks will require the group in some time.

Reply:
All checks of objects belonging to meeting rely on `assert_belongs_to_meeting`. Following the wiki it is obligatory for a temporary user to belong to a group. Having a group a temporary user would be always accepted to the meeting (of this group),  and there would be no difference to the normal user. But I wanted the user to be accepted with it's speciality having a meeting_id as temporary user.